data/ecosystem/vendors.yaml

```diff
@@ -349,3 +349,10 @@
   contact: 'otel@tingyun.com'
   oss: false
   commercial: true
+- name: VictoriaMetrics
+  distribution: false
+  nativeOTLP: false
```

Comment:
Hmm. Your docs would indicate you support direct OTLP ingest of metrics. In that case should this be`true`?